eng/common/scripts/get-codeowners.ps1

```diff
@@ -0,0 +1,119 @@
+param (
+  $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
+  $RootDirectory, # ideally $(Build.SourcesDirectory)
+  $AuthToken,
+  $VsoOwningUsers = "",
+  $VsoOwningTeams = "",
+  $VsoOwningLabels = ""
+)
+
+. (Join-Path ${PSScriptRoot} logging.ps1)
+
+$target = $TargetDirectory.ToLower().Trim("/")
+$codeOwnersLocation = Join-Path $RootDirectory -ChildPath ".github/CODEOWNERS"
+$ownedFolders = @{}
+
+if (!(Test-Path $codeOwnersLocation)) {
+  LogWarning "Unable to find CODEOWNERS file in target directory $RootDirectory"
+  exit 1
+}
+
+function VerifyAlias($APIUrl)
+{
+  if ($AuthToken) 
+  {
+    $headers = @{
+      Authorization = "bearer $AuthToken"
+    }
+  }
+  try
+  {
+    $response = Invoke-RestMethod -Headers $headers $APIUrl
+  }
+  catch 
+  {
+    LogDebug "Invoke-RestMethod ${APIUrl} failed with exception:`n$_"
+    LogDebug "This might be because a team alias was used for user API request or vice versa."
+    return $false
+  }
+  return $true
+}
+
+nuget install "CodeOwnersParser" -OutputDirectory . `
```

Comment:
@chidozieononiwu this doesn't look like the last committed state of this file. We never had a version that used the library like this. https://github.com/Azure/azure-sdk-tools/commit/4bfbe0075db68193a336ed4973df0505395069fd#diff-e3cfe80d53c2e2c455b3cc02c5592438031893d3e939b7abb57fefd94e734463 looks to be the last committed version.

Reply:
I have updated it.